src/controller/CHIPDeviceController.h

```diff
@@ -41,6 +41,7 @@
 #include <transport/SecureSessionMgr.h>
 #include <transport/TransportMgr.h>
 #include <transport/raw/UDP.h>
+#include <lib/mdns/Discovery_ImplPlatform.h>
```

Comment:
Could you use `Resolver.h` interface instead of the underlying implementation directly? Note that the existing Python command "resolve" uses the generic interface, so it's worth checking if a common Python interface for your and that commands can be created.

Reply:
Resolver is Operational discovery in fact, and this code is intended to do commission discovery as well as testing the PlatformImpl of mdns.